### ISSUE TITLE
Meetup during the week

### ISSUE THREAD

**Issue by blakmatrix** [nodeschool/seattle#19]
Hi all,

@louisnk and I are trying to get a more frequent meetup going. Question is: when is ideal?

Tuesdays look fairly open on meetup.com, only possible conflict I see is with new tech Seattle which is on every 3rd tuesday (though they say they do one ever 2nd or 3rd tuesday). Wednesday and thursdays seem too conflicting with other tech meetups. Friday, I doubt anyone would want to show up, and then there is Mondays which seems ok.

How does every 2nd Tuesday sound, or maybe we should use a voting software to try and come to a consensus? What are everyones thoughts?

**Comment by daveskull81** [comment]
I think Tuesday is a great day. I know that I could make it to those if it were on Tuesday nights.

**Comment by louisnk** [comment]
Tuesdays are probably the best day. Still waiting to hear back from Moz, but I think Tune might also be willing to host a ~2 hr session once/month. If there's still no word from Moz by Friday I'll email them again, and also reach out to Tune.

**Comment by blakmatrix** [comment]
@louisnk I chatted with Dudley yesterday over email only tangentially but NodeSchool stuff did come up and at first he thought the 28th was going to be at Moz which I assured him our hosting space for that day was already taken care of. Moz is definitely excited to host with us, he said "The 14th [of April] would likely be a better date than the 7th. Let's shoot for that. I'll let you and Emmi set that up."
